powersimdata/design/investment/create_mapping_files.py

```diff
@@ -156,18 +150,14 @@ def bus_to_reeds_reg(df):
 
 
 def bus_to_neem_reg(df):
-    """Given a dataframe of buses, return a dataframe of bus_id's with associated
-        NEEM region, lat, and lon of bus.
-    Used to map regional transmission investment cost multipliers.
-    Shapefile used to map is 'data/NEEM/NEEMregions.shp' which is pulled from Energy
-        Zones `Mapping tool <http://ezmt.anl.gov>`_. This map is overly detailed, so I
-        simplified the shapes using 1 km distance (Douglas-Peucker) method in QGIS.
-
-    :param pandas.DataFrame df: grid.bus instance.
-    :return: (*pandas.DataFrame*) -- bus_id map.
-        columns: bus_id, lat, lon, name_abbr (NEEM region)
-
-    Note: mapping may take a while, especially for many points.
+    """Map bus to ReEDS regions.
+
+    :param pandas.DataFrame df: bus data frame.
+    :return: (*pandas.DataFrame*) -- index: bus id, columns: lat, lon, name_abbr
+        (NEEM region)
+
+    .. note: the shapefile used for mapping is pulled from the Energy Zones `Mapping
+        tool <http://ezmt.anl.gov>`_. This map is overly detailed, so the shapes are simplified using 1 km distance (Douglas-Peucker) method in QGIS.
```

Comment:
Line too long?

Reply:
I don't think black will apply line length filter on doc strings. I used to leave it as it is or manually fix it.

Reply:
fixed